hack/markdownlint.sh

```diff
@@ -19,7 +19,7 @@ else
     --volume "${PWD}:/workdir:ro,z" \
     --entrypoint sh \
     --workdir /workdir \
-    registry.hub.docker.com/pipelinecomponents/markdownlint:latest \
+    registry.hub.docker.com/pipelinecomponents/markdownlint:0.12.0@sha256:0b8f9fcf0410257b2f3548f67ffe25934cfc9877a618b9f85afcf345a25804a2 \
```

Comment:
why do we need sha here? release version isnt enough?

Reply:
ok, +1, when we are configuring renovate for this, I hope renovate would understand this ?